### PR TITLE
Maintenance Day - Swap to theme imports

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Went from something like this: 
```
const Paragraph = styled.p(({ theme }) => ({
  color: theme.colors.gray700,
}))
```
to something like this: 

```
import {theme} from 'ui'

const Paragraph = styled.p({
  color: theme.colors.gray700,
})
```

## Justify why they are needed

As a team we talked about moving away from using the function syntax with Emotion and importing our theme directly from our ui package. This PR is a step in that direction.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/components/CheckList/CheckList.tsx [hunk 1]
```diff
@@ -17,11 +17,11 @@ export const Item = ({ children }: PropsWithChildren<unknown>) => {
   )
 }
 
-const StyledItem = styled.li(({ theme }) => ({
+const StyledItem = styled.li({
   display: 'flex',
   alignItems: 'center',
-  gap: theme.space[2],
-}))
+  gap: theme.space.xs,
```
Comment (amandaflod) [inline apps/store/src/components/CheckList/CheckList.tsx:23]: sneaky 🕵️‍♀️

**Review by amandaflod — APPROVED** [review]
Looks good to me!

File: apps/store/src/components/CheckList/CheckList.tsx (hunk 1 above)
Comment (KarnellSchultz) [reply apps/store/src/components/CheckList/CheckList.tsx:23]: Tried not to changes those 👀 

Maybe the next PR though???

**Review by gustaveen — APPROVED** [review]
Ouff! MVP of maintenance day right there! ⭐️

**Pull request merged**